src/manifest.ts

```diff
@@ -23,11 +23,13 @@ export class SlackManifest {
         // todo: is there a more idiomatic way of defining this? constant file?
         "major_version": 2,
       },
+
+      "function_runtime": this.getFunctionRuntime(),
```

Comment:
This currently introduces manifest validation issues as support for this isn't in the backend yet correct?

Reply:
Yes, currently` function_runtime` is under settings schema in the [ `webapp app_manifest`](https://slack-github.com/slack/webapp/blob/master/js/modern/generated/types/api/schemas/app-manifests/v1/settings/schema.ts). To be able to validate ` function_runtime` in our `ManifestSchema`, we also have to update the[ `webapp manifest.ts`](https://slack-github.com/slack/webapp/blob/master/js/modern/generated/types/api/schemas/app-manifests/v1/manifest.ts)

Reply:
Should we move `function_runtime` under settings here instead? (Keep it top level but move it under settings when generating the manifest)